src/bootstrap/overview-sheet-boostrapper.js

```diff
@@ -4,13 +4,23 @@ class OverviewSheetBootstrapper extends BaseSheetBootstrapper {
   }
 
   createSheet() {
+    const letterTransAmount = colToLetter(TransactionsSheet.columnAmmount);
+    const letterTransDate = colToLetter(TransactionsSheet.columnDate);
+    const letterTransIsInvestment = colToLetter(TransactionsSheet.columnIsInvestment);
+    const letterTransIsIncome = colToLetter(TransactionsSheet.columnIsIncome);
     const numRows = 200;
     const summaryData = [];
     for (let i = 0; i < MONTHS.length; i += 1) {
+      const formulaExpenses = `=sumifs('All-Transactions'!${letterTransAmount}:${letterTransAmount}, 'All-Transactions'!${letterTransIsIncome}:${letterTransIsIncome}, "No", 'All-Transactions'!${letterTransIsInvestment}:${letterTransIsInvestment}, "No", arrayformula(Month('All-Transactions'!${letterTransDate}:${letterTransDate})),${
+        i + 1
+      } )`;
+      const formulaIncome = `=sumifs('All-Transactions'!${letterTransAmount}:${letterTransAmount}, 'All-Transactions'!${letterTransIsIncome}:${letterTransIsIncome}, "Yes", 'All-Transactions'!${letterTransIsInvestment}:${letterTransIsInvestment}, "No", arrayformula(Month('All-Transactions'!${letterTransDate}:${letterTransDate})),${
```

Comment:
Income is showing as negative numbers. This is offsetting, let's make them positives

Reply:
done